source/cli/appservices-deploy-configure.txt

```diff
@@ -0,0 +1,87 @@
+.. _appservices-deploy-configure:
+
+============================
+appservices deploy configure
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Configure the settings for automatic deployment
+
+Walks you through configuring automatic deployment for your app
```

Comment:
do we want to mention here or on the deploy ref landing page that the CLI automatically enables auto deployment when you complete the interactive configuration?

Reply:
The content on these pages comes directly from the CLI code. Any changes we would make to these auto-generated pages would get overwritten. The engineering team would need to make that change in their codebase.